test/drbd.metrics

```diff
@@ -46,6 +46,10 @@ ha_cluster_drbd_upper_pending{resource="1-single-1",volume="0"} 1 1234
 # TYPE ha_cluster_drbd_lower_pending gauge
 ha_cluster_drbd_lower_pending{resource="1-single-0",volume="0"} 2 1234
 ha_cluster_drbd_lower_pending{resource="1-single-1",volume="0"} 2 1234
+# HELP ha_cluster_drbd_quorum Quorum status; 1 line per res, per volume
```

Comment:
@kermat  a `nit`: 

I think we could definitely improve the `help` description message perhaps.

I mean 1 line doesn't in which context? 

We could imho remove the 1 line. ( this would apply also for other metrics)

` HELP ha_cluster_drbd_quorum Quorum status per resource and per volume`
	
Something in this direction

Reply:
I agree. I was going to add some missing tests for the second fake resource in a separate PR to keep things tidy per diegoakechi's recommendation. So, I'll change the quorum description in this PR, and open another PR to address the missing tests and messages added in PR #106